test/utils.tsx

```diff
@@ -0,0 +1,40 @@
+import { i18n } from '@lingui/core'
```

Comment:
This file is stale and should be removed. It's been replaced with src/test/utils.